File: SumSquareDifference/SumSquareDifference.java
import java.util.concurrent.TimeUnit;

public class SumSquareDifference
{
  public static void main( String[] args )
  {
    int SumOfSquares = 0, SquareOfSum = 0;

    long startTime = System.nanoTime();
    for( int i = 1; i <= 100; i++ )
    {
      SumOfSquares += ( i * i );
      SquareOfSum += i;
    }
    SquareOfSum *= SquareOfSum;
    long totalTime = System.nanoTime() - startTime;

    System.out.println( "Difference: " + Integer.toString( SquareOfSum - SumOfSquares ) );
    System.out.println( "TotalTime: " + Long.toString( totalTime ) );
  }
}

File: 006_SumSquareDifference/SumSquareDifference.js
let sumOfSquares = 0, squareOfSum = 0;

for(let i = 1; i <= 100; i++){
  sumOfSquares += Math.pow(i, 2);
  squareOfSum += i;
}

squareOfSum = Math.pow(squareOfSum, 2);
console.log(`${squareOfSum - sumOfSquares}`);